app/main/views/services.py

```diff
@@ -181,7 +181,7 @@ def import_service(service_id):
     """
     is_valid_service_id_or_400(service_id)
 
-    now = datetime.now()
+    now = datetime.utcnow()
```

Comment:
There are a couple more of these in the tests, it might be nice to have them all using `utcnow`, although obviously doesn't make a lot of difference in tests.